packages/native-ui/src/config/theme/ListItem.ts

```diff
@@ -0,0 +1,168 @@
+import { createStyle } from '@gluestack-style/react';
+
+export const ListItem = createStyle({
+  padding: '$4',
+  flexDirection: 'row',
+  gap: '$3',
+
+  _icon: {
+    color: '$grey800',
+  },
+
+  _supportingText: {
+    color: '$grey600',
+    props: {
+      lineHeight: '$sm',
+    },
+  },
+
+  _leadingContent: {
+    _icon: {
+      color: '$grey800',
+    },
+  },
+
+  _trailingContent: {
+    _icon: {
+      color: '$grey800',
+    },
+    _trailingIcon: {
+      color: '$grey800',
+    },
+  },
+
+  ':disabled': {
+    _text: {
+      color: '$grey400',
+    },
+    _icon: {
+      color: '$grey400',
+    },
+    _supportingText: {
+      color: '$grey400',
+    },
+    _leadingContent: {
+      _icon: {
+        color: '$grey400',
+      },
+    },
+    _trailingContent: {
+      _icon: {
+        color: '$grey400',
+      },
+      _trailingIcon: {
+        color: '$grey400',
+      },
+    },
+  },
+
+  _dark: {
+    _icon: {
+      color: '$darkGrey800',
+    },
+
+    _supportingText: {
+      color: '$darkGrey600',
+    },
+
+    _leadingContent: {
+      _icon: {
+        color: '$darkGrey800',
+      },
+    },
+
+    _trailingContent: {
+      _icon: {
+        color: '$darkGrey800',
+      },
+      _trailingIcon: {
+        color: '$darkGrey800',
+      },
+    },
+
+    ':disabled': {
+      _text: {
+        color: '$darkGrey500',
+      },
+      _icon: {
+        color: '$darkGrey500',
+      },
+      _supportingText: {
+        color: '$darkGrey500',
+      },
+      _leadingContent: {
+        _icon: {
+          color: '$darkGrey500',
+        },
+      },
+      _trailingContent: {
+        _icon: {
+          color: '$darkGrey500',
+        },
+        _trailingIcon: {
+          color: '$darkGrey500',
+        },
+      },
+    },
+  },
+  variants: {
+    divider: {
```

Comment:
@jordmccord need for aligment here 👇 
In Figma the divider is the`Divider` component which has it's built in custom props and customisations. 

<img width="962" alt="Screenshot 2024-08-27 at 13 47 36" src="https://github.com/user-attachments/assets/7c0fda19-eccf-44b1-a2c2-79442c9f2699">

instead of using a border for the `List Item`, we should flag all dependencies and create tasks for them 🙏  before we create the component. Using a `Divider` component preserves flexibility and maintains a connection to the actual component. If a designer modifies the `Divider` props in a `List`, the engineer can directly change those props instead of overriding these "divider" styles.

Let's ensure this isn't a blocker for release in our app ui alignments. However, we'll need to update the dependency when the `Divider` becomes a code library component in code and create follow-up tasks accordingly to replace this.